test/release/examples/primers/FFTWlib.chpl

```diff
@@ -10,14 +10,14 @@
 
   .. code-block:: text
 
-    chpl testFFTW.chpl
+    chpl FFTWlib.chpl
 
   Otherwise, use the following (where ``$FFTW_DIR`` points to your
   FFTW installation):
 
   .. code-block:: text
 
-    chpl testFFTW.chpl -I$FFTW_DIR/include -L$FFTW_DIR/lib
+    chpl FFTWlib.chpl -I$FFTW_DIR/include -L$FFTW_DIR/lib
```

Comment:
Thanks for picking this up!  Further down in the file, there's a reference to `testFFTW.good` that also should be updated.